.misc/deploy.pypi.sh

```diff
@@ -11,11 +11,16 @@ echo "Uploading coala to pypi"
 pip3 install twine wheel
 python3 setup.py sdist bdist_wheel
 # Upload one by one to avoid timeout
-# Don't block merging because pypi is in error 500 mood again
-set +e
-twine upload dist/*.whl -u "$PYPIUSER" -p "$PYPIPW"
-twine upload dist/*.tar.gz -u "$PYPIUSER" -p "$PYPIPW"
-set -e
+twine upload dist/* -u "$PYPIUSER" -p "$PYPIPW" 2>&1 | tee twine_output.txt
+if [ "${PIPESTATUS[0]}" -ne 0 ]; then
+    SEARCH_STR="500 Server Error"
+    grep -q "$SEARCH_STR" twine_output.txt
+    if [ "$?" -eq 0 ]; then
```

Comment:
I think you could directly inline the command:
```
if grep -q "$SEARCH_STR" twine_output.txt; then
...
```
as this happens on my bash terminal:
```
 mak  ~  if echo x; then
           echo y
          else
           echo z
          fi
x
y
```